ci-operator/step-registry/baremetalds/e2e/ovn/debug/enable-console/baremetalds-e2e-ovn-debug-enable-console-commands.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+set -x
+
+if [ "${ENABLE_DEBUG_CONSOLE:-}" != "true" ]; then
+  exit 0
+fi
+
+# Fetch packet basic configuration
+# shellcheck source=/dev/null
+source "${SHARED_DIR}/packet-conf.sh"
+
+set +x
+PASSWD=$(</dev/random tr -cd '[:alnum:]' | fold -w 10 | head -1 || true)
+echo "PASSWD=$PASSWD" > "${SHARED_DIR}"/console.passwd
+set -x
+
+# shellcheck disable=SC2087
+ssh "${SSHOPTS[@]}" "root@${IP}" bash - <<EOF
+source ~/dev-scripts-additional-config
+nodes=\$(oc get node -o jsonpath='{.items[*].metadata.name}')
+
+for node in \$nodes; do
+  oc debug node/"\$node" -- chroot /host bash -c "rpm-ostree kargs --append-if-missing=console=ttyS0,115200n8"
+done
+
+oc adm reboot-machine-config-pool mcp/worker mcp/master
+oc adm wait-for-node-reboot nodes --all
+
+for node in \$nodes; do
+  oc debug node/"\$node" -- chroot /host bash -c "echo $PASSWD | passwd core --stdin"
+  # preload toolbox image to have it available on disconnected nodes
+  oc debug node/"\$node" -- chroot /host toolbox exit
```

Comment:
I am guessing the toolbox image will not get garbage collected?

Reply:
I don't know. I am not aware of anything that garbage collects podman images. Is that a thing?